resources/queries/targetedms/librarymoleculeprecursor/LibraryPrecursors.qview.xml

```diff
@@ -0,0 +1,28 @@
+<customView xmlns="http://labkey.org/data/xml/queryCustomView" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        label="Library Precursors">
+    <columns>
+        <column name="MoleculeId/PeptideGroupId/RunId/File" />
+        <column name="MoleculeId/MoleculeName" />
+        <column name="IonFormula" />
+        <column name="CustomIonName" />
+        <column name="Charge" />
+        <column name="IsotopeLabelId" />
+        <column name="Mz" />
+        <column name="MassMonoisotopic" />
+        <column name="MassAverage" />
+        <column name="CollisionEnergy" />
+        <column name="DeclusteringPotential" />
+        <column name="NoteAnnotations" />
+    </columns>
+
+    <filters>
+        <filter column="RepresentativeDataState" operator="eq" value="1"/>
+    </filters>
+
+    <sorts>
+        <sort column="IonFormula" />
```

Comment:
Can we sort on the MoleculeId/MoleculeName here instead of IonFormula and CustomIonName both of which can be null?

Reply:
Added MoleculeName as primary sort, but left the others as secondary ordering